core/server/data/migrations/versions/3.23/01-migrate-bulk-email-settings.js

```diff
@@ -0,0 +1,64 @@
+const logging = require('../../../../../shared/logging');
+
+module.exports = {
+    config: {
+        transaction: true
+    },
+
+    async up({transacting: knex}) {
+        const defaultValue = {
+            provider: 'mailgun',
+            apiKey: '',
+            domain: '',
+            baseUrl: ''
```

Comment:
Should these be `null` rather than `''` to match `default-settings.json`?

Reply:
Yes good point :+1: updating now